### PR TITLE
Clean up some text

### PR DESCRIPTION
**Pull request by mariusandra**
Some small changes:
- update the display name for maxmind and currency plugins
- remove the extra images (they're in the repos now)
- other random tweaks

### CODE REVIEW

**Comment by mariusandra** [comment]
I reverted one change and added back images for the npm plugins.

**Comment by yakkomajuri** [comment]
Yeah was about to say - the images were for npm plugins

File: repository.json [hunk 1]
```diff
@@ -71,8 +72,7 @@
     "verified": true,
     "maintainer": "official",
     "displayOnWebsiteLib": true,
-    "type": "ingestion_filtering",
-    "imageLink": "https://raw.githubusercontent.com/PostHog/plugin-repository/main/logos/schema-enforcer.png"
```
Comment (yakkomajuri) [inline repository.json:75]: schema enforcer was left without a logo.

handy url:

https://raw.githubusercontent.com/PostHog/posthog-schema-enforcer-plugin/main/logo.png

**Comment by yakkomajuri** [comment]
Otherwise this is good

**Pull request merged**